index.html

```diff
@@ -0,0 +1,339 @@
+            alt="Tonic Company"
+            class="tonic-image"
+          />
+        </div>
+      </section>
+      <!--Work Third Figma-->
+      <section class="card">
+        <div class="card-box">
+          <img
+            src="Asset/Snapshoot Portfolio2.png"
+            alt="Tonic Company"
+            class="tonic-image"
+          />
+        </div>
+        <div class="card-box">
+          <h1 class="TonicHeader">Facebook 360</h1>
+          <div class="HeaderTitle">
+            <h2 class="tonic-title">CANOPY</h2>
+            <img src="Asset/Counter.png" class="counter" />
+            <h3 class="TonicEnd">Full Stack Dev</h3>
+            <img src="Asset/Counter.png" class="counter" />
+            <h3 class="TonicEnd">2015</h3>
+          </div>
+          <p class="Content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="skill-box">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <input type="submit" class="btnResume" value="See Project" />
+        </div>
+      </section>
+      <!--Work Section Fourth Figma-->
+      <section class="card">
+        <div class="card-box">
+          <h1 class="TonicHeader">Uber Navigation</h1>
+          <div class="HeaderTitle">
+            <h2 class="tonic-title">Uber</h2>
+            <img src="Asset/Counter.png" class="counter" />
+            <h3 class="TonicEnd">Lead Developer</h3>
+            <img src="Asset/Counter.png" class="counter" />
+            <h3 class="TonicEnd">2015</h3>
+          </div>
+          <p class="Content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="skill-box">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <input type="submit" class="btnResume" value="See Project" />
+        </div>
+        <div class="card-box">
+          <img
+            src="Asset/Snapshoot Portfolio3.png"
+            alt="Tonic Company"
+            class="tonic-image"
+          />
+        </div>
+      </section>
+      <div id="about-contact">
+        <section class="about-meheader" id="about">
+          <div class="card-box">
+            <h1 class="about-me">About Myself!</h1>
+            <p class="AboutContent">
+              Hello I'm a software developer! I can help you build a product,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don't
+              hestiate to contact me.
+            </p>
+            <h2 class="connect">LET'S CONNECT</h2>
+            <ul class="social">
+              <li>
+                <a href=""><img src="Asset/vector1git.png" alt="" /></a>
+              </li>
+              <li>
```

Comment:
- [ ] Kindly fix the following descripancies,

- Figma design
<img width="368" alt="image" src="https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/41163571/1f30ce56-0bba-4e0a-b751-40b81d8d5065">

- Your design
<img width="311" alt="image" src="https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/41163571/38700660-5915-4ee1-893d-931cf81539bf">

- The languages icon should be in row.
- The frameworks and skills, section should have more height and width just like the design
- The LET"S CONNECT button font-size should be reduced, while the icons should have more spacing and the margin-left should be effected.
- The Get my resume button should have lesser padding to the right and left.
- The whole section should have more padding to the bottom, top, left and right.
- The top right corner should have curvy border radius.